src/ngraph/runtime/executable.cpp

```diff
@@ -130,25 +130,26 @@ void runtime::Executable::save(std::ostream& /* output_stream */)
     throw runtime_error("save operation unimplemented.");
 }
 
-shared_ptr<runtime::Tensor> runtime::Executable::create_input_tensor(size_t /* input_index */)
+shared_ptr<runtime::Tensor> runtime::Executable::create_input_tensor(size_t /* input_index */,
```

Comment:
Why aren't these defined as `=0` so they are detected as not implemented at compile-time?

Reply:
this PR just reuses existing API, if we make it pure virtual this would be a breaking change for other backends.

Reply:
@diyessi These were never required methods. I think in the future we probably want to switch to a single API for creating tensors but this is just a step along the way.

Reply:
@ashokei I'm pretty sure what you have will break the NNP backend where we are already overloading old API which has now been removed. We will get compile errors with this code.

Reply:
will make second API,  with a default implementation.